### ISSUE TITLE
PDF preview

### ISSUE THREAD

**Issue by quadur** [paginagmbh/redmine_lightbox2#10]
After clicking the preview icon along the pdf the lighbox popup shows but its an empty grey area. Image previews work fine.

I'm testing it on a clean install of redmine 3.0
![pdf](https://cloud.githubusercontent.com/assets/5623268/7881317/002845f8-0602-11e5-9473-71c17c98fa1f.png)

**Comment by tofi86** [comment]
Well, there has been an issue recently (#4), but it's resolved. Make sure you use the latest version of the plugin.

The display of PDF attachments also depends on your browser settings.
In Firefox for example, make sure the PDF doesn't get auto-opened in Adobe Reader or other programs. You can check your settings for that:

![bildschirmfoto 2015-05-29 um 13 15 17](https://cloud.githubusercontent.com/assets/365118/7881584/de488328-0604-11e5-99a5-3f30b3b5419b.png)

**Comment by quadur** [comment]
Just checked: git log states: commit 005ace4d7e4c568a615b92db91342785d7f89a78 and it seems to be the latest.

The problem exists in firefox and chrome (both have the 'preview in browser') setting for pdfs.

**Comment by tofi86** [comment]
which redmine version do you use? 3.0.0 or 3.0.3?

**Comment by quadur** [comment]
Redmine version                3.0.3.stable.14274
  Ruby version                   2.2.1-p85 (2015-02-26) [x86_64-linux]
  Rails version                  4.2.1

**Comment by tofi86** [comment]
Well, I did another test on my dev installation and this is what I observed:

I had PDF's which were recognized by redmine as `application/x-pdf` which couldn't be displayed inline and for which I couldn't even activate "preview in firefox" option in the settings.

![bildschirmfoto 2015-05-29 um 14 09 43](https://cloud.githubusercontent.com/assets/365118/7882291/78369cc0-060c-11e5-98ab-22a16fabe58f.png)

Than I had PDF's which were recognized by redmine as `application/pdf` – as they should – and which displayed inline perfectly.

![bildschirmfoto 2015-05-29 um 14 09 57](https://cloud.githubusercontent.com/assets/365118/7882286/7066f738-060c-11e5-9db9-1a3b2cb360ae.png)

This is my firefox settings page. Maybe the first represents mimetype `application/pdf` and the latter `application/x-pdf` – for which "preview in firefox" isn't available...
